src/chi/components/tables/tables.scss

```diff
@@ -146,6 +169,101 @@ th {
     }
   }
 
+  &.-inverse {
+    thead {
+      tr {
```

Comment:
Need to define a background color on the tr as it's inheriting white from `.chi .chi-table tr`